tools/docker/.screenrc

```diff
@@ -0,0 +1 @@
+shell "/bin/bash"
```

Comment:
I'm quite fond of [fish](https://de.wikipedia.org/wiki/Fish_(Shell)). Could we use it here?

Reply:
I don't see any problem, could install fish as extra package just like screen. I'll give it a try.

Reply:
Changed the screen shell to fish now (rebased/melded changes).

```
Welcome to fish, the friendly interactive shell
root@252a62dfbf2e /iceoryx# ./build/iceoryx_examples/icedelivery/ice_sender
Sending: 0
Sending: 1
Sending: 2
Sending: 3
```